webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -139,4 +141,18 @@ export const getContentViewVersions = (cvId, params) => {
   });
 };
 
+export const pollPublishTask = (cvPublishVersionKey, task) => async (dispatch) => {
+  dispatch(startPollingTask(cvPublishVersionKey, task));
+};
+
+export const handleFinishedTask = (cvId, cvPublishVersionKey, task, pending = false) =>
```

Comment:
This has enough args to make it confusing.  Probably better to use an object and destructure.
```suggestion
export const handleFinishedTask = ({cvId, cvPublishVersionKey, task, pending: false}) =>
```

Reply:
Updated to use an object instead of separate params.